README.md

```diff
@@ -52,6 +52,7 @@ QGIS is a full-featured, user-friendly, free-and-open-source (FOSS) geographical
 ### 2. Beautiful cartography
 - Large variety of rendering options in 2D and 3D
 - Fine control over symbology, labeling, legends and additional graphical elements for beautifully rendered maps
+- Near-complete replication (and significant extension) of symbology options available in proprietary software by ESRI
```

Comment:
This sounds like the options to replicate the symbology are available in an esri program.

Reply:
Added 2 words to avoid that interpretation, thanks.